src/ray/raylet/node_manager.cc

```diff
@@ -717,7 +722,7 @@ void NodeManager::HandleReleaseUnusedBundles(
   }
 
   // Return unused bundle resources.
-  placement_group_resource_manager_->ReturnUnusedBundle(in_use_bundles);
+  placement_group_resource_manager_->ReturnUnusedBundle(in_use_bundles_flat);
```

Comment:
Don't we need to exlucde bundle index == -1?

Reply:
Good point. Let me revisit this.